.github/workflows/patch-images.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  pull_request:
+    branches:
+      - main
+
+jobs:    
+  patch: 
+      runs-on: ubuntu-latest
+      strategy:
+        fail-fast: false
+        matrix:
+          images: ['ghcr.io/azure/azure-workload-identity/proxy-init:latest-linux-arm64', 'ghcr.io/azure/azure-workload-identity/proxy-init:latest-linux-amd64']
+      steps:
+      - name: Generate Trivy Report
+        uses: aquasecurity/trivy-action@41f05d9ecffa2ed3f1580af306000f734b733e54
+        with:
+          scan-type: 'image'
+          format: 'json'
+          output: 'report.json'
+          ignore-unfixed: true
+          vuln-type: 'os'
+          image-ref:  ${{ matrix.images }}
+      - name: Check Vuln Count
+        id: vuln_cout
+        run: |
+          report_file="report.json"
+          vuln_count=$(jq '.Results | length' "$report_file")
+          echo "vuln_count=$vuln_count" >> $GITHUB_OUTPUT
+      - name: Copa Action
+        if: steps.vuln_cout.outputs.vuln_count != '0'
+        id: copa
+        uses: project-copacetic/copa-action@v1.0.0
```

Comment:
copacetic-action now i think?

please add hash and version too

Reply:
I think it needs to be referenced by the repo name instead of action name